plugins/woocommerce-blocks/tests/e2e/tests/all-reviews/all-reviews.block_theme.spec.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { expect, test } from '@woocommerce/e2e-playwright-utils';
+
+const blockData = {
+	name: 'woocommerce/all-reviews',
+	selectors: {
+		frontend: {
+			firstReview:
+				'.wc-block-review-list-item__item:first-child .wc-block-review-list-item__text p',
+		},
+	},
+};
+
+test.describe( `${ blockData.name } Block`, () => {
+	test( 'renders a review in the editor and the frontend', async ( {
+		admin,
+		editor,
+		page,
+		editorUtils,
+	} ) => {
+		await admin.createNewPost();
+		await editor.insertBlock( { name: blockData.name } );
+
+		await expect( page.getByText( 'Nice album!' ) ).toBeVisible();
```

Comment:
Personally, I'd like to see these strings refactored somewhere else, since they get repeated a bunch and are tied to the specific shape of the fixture. What do you think?

Reply:
Good point, in d33455405544139a270558745a5b939e1a8f41d0 I added a `reviews` variable in `data.ts` so we can reuse it across tests. Unfortunately, I didn't find an easy way to share the data with the setup script, so there is still some duplication between `reviews.sh` and `data.ts`. Happy to make further modifications if you have any ideas on how to improve that!

Reply:
Thank you so much for following my suggestions! Regarding this question, if we wanted to go full one-source-of-truth, we could refactor this data in a JSON file, and then use something like `jq` (https://cameronnokes.com/blog/working-with-json-in-bash-using-jq/) to read the data and parse it out.

Maybe this is long-term the way to go with all our fixture-generating code, to be honest. To clean it up.

Right now, however, might be a bit overkill maybe? The only concern I have is for future developers when they peruse the code. So, could I perhaps suggest that before your `export const reviews` you add something like:

```js
// source: plugins/woocommerce-blocks/tests/e2e/bin/scripts/parallel/reviews.sh
```

So that people doing changes (or just exploring) know where these strings come from. What do you think?

Reply:
Good idea! Done in ee863d81ab677ed9f20387cbf27a8dd1d09b137f.

> Right now, however, might be a bit overkill maybe?

Yup, agree. I think this is something we can iterate on in the future if we see it is needed in more test data.